code/modules/research/designs/bio_devices.dm

```diff
@@ -1,51 +1,49 @@
 /datum/prototype/design/science/biotech
 	abstract_type = /datum/prototype/design/science/biotech
+	category = DESIGN_CATEGORY_MEDICAL
 	materials_base = list(MAT_STEEL = 30, MAT_GLASS = 20)
 
-/datum/prototype/design/science/biotech/generate_name(template)
-	return "Biotech device prototype ([..()])"
-
 // Biotech of various types
 
 /datum/prototype/design/science/biotech/mass_spectrometer
 	desc = "A device for analyzing chemicals in blood."
-	id = "mass_spectrometer"
+	id = "RNDDesignMassSpectrometer"
```

Comment:
going forwards we should also not (this doesn't mean you have to rename this in this PR that's not a reasonable ask at this stage tbh and we have a bit before persistence starts and this matters) prefix with `RND` or stuff like that, as there'll basically just be generic designs and corporate designs